### PR TITLE
docs(example): update the README in examples

### PR DESCRIPTION
**Pull request by xiaojian-hong**
This change is [Reviewable](https://reviewable.io/reviews/yomorun/yomo/285)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,7 @@ vendor/
 main
```
Comment (fanweixiao) [inline .gitignore:19]: Suggest build file to `/bin` directory, then we can add `bin/` to `.gitignore`

File: README_CN.md [hunk 2]
```diff
@@ -150,9 +158,9 @@ $ yomo dev
 
 ## 📚 Documentation
 
-+ `YoMo-Source`: [yomo.run/source](https://docs.yomo.run/source)
-+ `YoMo-Stream-Function` (formerly flow): [yomo.run/stream-function](https://docs.yomo.run/stream-function)
-+ `YoMo-Zipper`: [yomo.run/zipper](https://docs.yomo.run/zipper)
++ `YoMo-Source`: [docs.yomo.run/source](https://docs.yomo.run/source)
++ `YoMo-Stream-Function`: [docs.yomo.run/stream-function](https://docs.yomo.run/stream-function)
```
Comment (venjiang) [inline README_CN.md:162]: should be [docs.yomo.run/stream-fn](https://docs.yomo.run/stream-fn)

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- [basic](https://github.com/yomorun/yomo/tree/master/example/basic): process the streams from IoT sound sensor.
+
+## Multiple stream functions
+
+- [multi-stream-fn](https://github.com/yomorun/yomo/tree/master/example/multi-stream-fn): use 3 stream functions to process the streams in different cases.
+  - stream-fn-1: calculate the sound value in real-time.
+  - stream-fn-2: print the warning message when the sound value reaches a threshold.
+  - stream-fn-3: calculate the average value in a sliding window.
+
+## Multiple zippers
```
Comment (fanweixiao) [inline example/README.md:19]: shall we rename this as `Cascading zippers`?

File: README.md [hunk 4]
```diff
@@ -161,9 +169,9 @@ Congratulations! You have done your first YoMo Stream Function.
 
 ## 📚 Documentation
 
-+ `YoMo-Source`: [yomo.run/source](https://docs.yomo.run/source)
-+ `YoMo-Stream-Function` (formerly flow): [yomo.run/stream-function](https://docs.yomo.run/stream-function)
-+ `YoMo-Zipper`: [yomo.run/zipper](https://docs.yomo.run/zipper)
++ `YoMo-Source`: [docs.yomo.run/source](https://docs.yomo.run/source)
++ `YoMo-Stream-Function`: [docs.yomo.run/stream-function](https://docs.yomo.run/stream-function)
```
Comment (venjiang) [inline README.md:173]: should be [docs.yomo.run/stream-fn](https://docs.yomo.run/stream-fn)

File: example/1-pipeline/README.md [hunk 5]
```diff
@@ -16,12 +16,25 @@ By introducting [YoMo](https://yomo.run), we can build it just like `unix pipeli
 
 ## 1. Prepare
 
-Install yomo cli `go install github.com/yomorun/cli/yomo@latest`:
+Install YoMo CLI
+
+### Binary (Recommended)
+
+```bash
+Ξ _wrk/yomo git:(feat/example) ▶ curl -fsSL "https://bina.egoist.sh/yomorun/cli?name=yomo" | sh
```
Comment (venjiang) [inline example/1-pipeline/README.md:24]: `Ξ _wrk/yomo git:(feat/example) ▶` should be removed.

**Review by fanweixiao — COMMENTED** [review]
I will try to update in this PR

File: example/1-pipeline/README.md [hunk 6]
```diff
@@ -16,12 +16,25 @@ By introducting [YoMo](https://yomo.run), we can build it just like `unix pipeli
...
+  ==> Resolved version latest to v0.1.7
+  ==> Downloading asset for darwin amd64
+  ==> Installing yomo to /usr/local/bin
+  ==> Installation complete
+```
+
+### Or build from source
 
 ```bash
 Ξ _wrk/yomo git:(feat/example) ▶ go install github.com/yomorun/cli/yomo@latest
```
Comment (venjiang) [inline example/1-pipeline/README.md:35]: `Ξ _wrk/yomo git:(feat/example) ▶` should be removed.

File: example/2-iopipe/README.md [hunk 7]
```diff
@@ -16,12 +16,25 @@ By introducting [YoMo](https://yomo.run), we can build it just like `unix pipeli
...
+  ==> Resolved version latest to v0.1.7
+  ==> Downloading asset for darwin amd64
+  ==> Installing yomo to /usr/local/bin
+  ==> Installation complete
+```
+
+### Or build from source
 
 ```bash
 Ξ _wrk/yomo git:(feat/example) ▶ go install github.com/yomorun/cli/yomo@latest
```
Comment (venjiang) [inline example/2-iopipe/README.md:35]: `Ξ _wrk/yomo git:(feat/example) ▶` should be removed.

**Review by venjiang — COMMENTED** [review]
docs.yomo.run/stream-function -> docs.yomo.run/stream-fn
`Ξ _wrk/yomo git:(feat/example) ▶` should be remove

File: example/README.md (hunk 3 above)
Comment (fanweixiao) [reply example/README.md:19]: done

File: README_CN.md (hunk 2 above)
Comment (fanweixiao) [reply README_CN.md:162]: done

**Review by Yoname — APPROVED** [review]

**Review by fanweixiao — APPROVED** [review]

**Pull request merged**